src/Common/MorganStanley.ComposeUI.Common.VisualUtils/MorganStanley.ComposeUI.Common.VisualUtils.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd name this project differently as it is Avalonia-specific. Maybe we should group packages by UI technology and put them into respective folders.

Reply:
Yes, correct, I'll rename it to make sure that it is clearly marked as Avalonia project.

Reply:
the avalonia related reusable projects are now in their own folders Visuals/Avalonia and have 'Avalonia' as part of their names. Resolving this issue.